mobile/src/main/java/org/openhab/habdroid/ui/PreferencesActivity.kt

```diff
@@ -396,6 +398,15 @@ class PreferencesActivity : AbstractBaseActivity() {
                 preferenceScreen.removePreferenceRecursively(PrefKeys.TASKER_PLUGIN_ENABLED)
             }
 
+            if (Util.isFlavorFoss) {
+                preferenceScreen.removePreference(crashReporting)
+            } else {
+                crashReporting.setOnPreferenceClickListener {
+                    CrashReportingHelper.initialize(requireActivity().application)
```

Comment:
AFAIK this is called before the changed value is saved.

Reply:
It's actually the other way around; `setOnPreferenceChangeListener` is called before the change is committed, `setOnPreferenceClickListener` after.